src/utils/helpers/structure/buildSectionsStructureNEW.ts

```diff
@@ -33,7 +33,7 @@ export const buildSectionsStructure = ({
       // Will build the array of elements
       const state = elements.map((element) => ({ element }))
 
-      const pageName = `Page ${pageNumber}`
+      const pageName = `Page${pageNumber}`
```

Comment:
This is currently used to display the pages in the UI. To use for passing to props in to other functions you can use the Page property (in the object) `number` which has the page number to use in the URL.

Reply:
Ok I'll change

Reply:
8ff8640fef745374f34fcf13205a793558205b09